js/modules/cohortbuilder/components/CohortExpressionEditorTemplate.html

```diff
@@ -99,7 +99,8 @@
 		</div>
 		<div data-bind="eventListener: [
 			 { event: 'click', selector: '.copyInclusionRule', callback: copyInclusionRule},
-			 { event: 'click', selector: '.deleteInclusionRule', callback: deleteInclusionRule}]">
+			 { event: 'click', selector: '.deleteInclusionRule', callback: deleteInclusionRule},
+			 { event: 'click', selector: '.addConceptSet', callback: addConceptSet}]">
```

Comment:
This event handler for .addConceptSet selector: is it necessary?  The callback function doesn't actually do anything, so I'm wondering why this was added?